app/javascript/src/components/Reports/outstandingInvoices/Container/index.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { Fragment } from "react";
+import TotalHeader from "common/TotalHeader";
+import { useEntry } from "components/Reports/context/EntryContext";
+import { cashFormatter } from "helpers/cashFormater";
+import { currencySymbol } from "helpers/currencySymbol";
+import TableRow from "./TableRow";
+import { ClientList } from "../interface";
+
+const TableHeader = () => (
+  <thead>
+    <tr className="flex flex-row items-center">
+      <th
+        scope="col"
+        className="w-2/5 pr-0 py-5 text-left text-xs font-normal text-miru-dark-purple-600 tracking-widest"
+      >
+        CLIENT /
+        <br />
+        INVOICE NO.
+      </th>
+      <th
+        scope="col"
+        className="w-1/5 px-3 py-5 text-left text-xs font-normal text-miru-dark-purple-600 tracking-widest"
+      >
+        Issued Date /
+        <br />
+        Due date
+      </th>
+      <th
+        scope="col"
+        className="w-1/5 px-7 py-5 text-center text-xs font-normal text-miru-dark-purple-600 tracking-widest"
+      >
+        AMOUNT
+      </th>
+      <th
+        scope="col"
+        className="w-1/5 pl-6 py-5 text-right text-xs font-normal text-miru-dark-purple-600 tracking-widest"
+      >
+        STATUS
+      </th>
+    </tr>
+  </thead>
+);
+
+const Container = () => {
+  const { outstandingOverdueInvoice } = useEntry();
+
+  const currencySymb = currencySymbol(outstandingOverdueInvoice.currency);
+
+  return (
+    <Fragment>
+      <TotalHeader
+        firstTitle={"TOTAL OUTSTANDING"}
+        firstAmount={`${currencySymb}${cashFormatter(outstandingOverdueInvoice.summary.totalOutstandingAmount)}`}
+        secondTitle={"TOTAL OVERDUE"}
+        secondAmount={`${currencySymb}${cashFormatter(outstandingOverdueInvoice.summary.totalOverdueAmount)}`}
```

Comment:
We can dry out this Amount format to a function
```suggestion
        secondAmount={`formattedAmount(outstandingOverdueInvoice.summary.totalOverdueAmount)`}
```
and formattedAmount can be something similar to this
```
const formattedAmount = (amount) => `${currencySymb}${cashFormatter(amount)}`
 
```

Reply:
Here, `formattedAmount` will give us amount in number format, but `cashFormatter` will give us amount in "K" format if it is in thousand range.